### PR TITLE
📝 add documentation to CMake find-package support

### PR DESCRIPTION
**Pull request by guylevy84**
This PR adds documentation in README.md to CMake find-package support (#645)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,6 +96,23 @@ That should do it!
...
+#### find_package():
+If you prefer not to use `fetch_content`, you can download, build, and install the library and then use CMake `find_package()` function to integrate it into a project.
+
+**Note:** this feature is feasible only if CPR_USE_SYSTEM_CURL is set. (see [#645](https://github.com/libcpr/cpr/pull/645))
+```Bash
+$ git clone https://github.com/libcpr/cpr.git
+$ cd cpr && mkdir build && cd build
+$ cmake .. -DCPR_USE_SYSTEM_CURL=ON
+$ make -j
+$ sudo make install
```
Comment (COM8) [inline README.md:108]: I prefer the CMake way of doing this, which is platform independent:
```suggestion
$ cmake --build .
$ sudo cmake --build . --target install
```

**Review by COM8 — CHANGES_REQUESTED** [review]
Thanks for your PR! Only one minor thing and we are ready to do.

**Comment by guylevy84** [comment]
@COM8, since CMake version >= 3.15 we can do `sudo cmake --install .` instead of `sudo cmake --build . --target install`

**Review by COM8 — APPROVED** [review]
Good point. Thanks!

**Pull request merged**